utils/config.go

```diff
@@ -43,6 +43,11 @@ type SSOSetting struct {
 	UserApiEndpoint string
 }
 
+type SSOSettings struct {
+	AllowEmailSignUp bool
```

Comment:
Is this really an SSO setting? Someone might want to disable signup for other reasons.